pkg/diagnostics/service_monitoring.go

```diff
@@ -12,16 +14,20 @@ import (
 
 // Tag keys.
 var (
-	componentKey      = tag.MustNewKey("component")
-	failReasonKey     = tag.MustNewKey("reason")
-	operationKey      = tag.MustNewKey("operation")
-	actorTypeKey      = tag.MustNewKey("actor_type")
-	trustDomainKey    = tag.MustNewKey("trustDomain")
-	namespaceKey      = tag.MustNewKey("namespace")
-	policyActionKey   = tag.MustNewKey("policyAction")
-	resiliencyNameKey = tag.MustNewKey("name")
-	policyKey         = tag.MustNewKey("policy")
-	componentNameKey  = tag.MustNewKey("componentName")
+	componentKey        = tag.MustNewKey("component")
+	failReasonKey       = tag.MustNewKey("reason")
+	operationKey        = tag.MustNewKey("operation")
+	actorTypeKey        = tag.MustNewKey("actor_type")
+	trustDomainKey      = tag.MustNewKey("trustDomain")
+	namespaceKey        = tag.MustNewKey("namespace")
+	policyActionKey     = tag.MustNewKey("policyAction")
```

Comment:
Maybe unrelated, but this and "trustDomain" are the only ones that use camelCase, while the others use snake_case... Should we standardize? (Although that may be a breaking change)

Reply:
although I agree standardizing these labels makes a lot of sense, I would consider renaming the label a breaking change and therefore it'd need to be a case of adding a new label and deprecating this one over a few releases. Something handled outside of the scope of this PR anyway.